content/sensu-go/6.4/operations/maintain-sensu/upgrade.md

```diff
@@ -11,25 +11,64 @@ menu:
     parent: maintain-sensu
 ---
 
-To upgrade to the latest version of Sensu Go, [install the latest packages][1] and restart the services.
+To upgrade to the latest version of Sensu Go:
 
-{{% notice note %}}
-**NOTE**: For systems that use `systemd`, run `sudo systemctl daemon-reload` before restarting the services.
-{{% /notice %}}
+1. [Install][1] the latest packages or Docker image.
 
-To restart the Sensu agent, run:
+2. For systems that use `systemd`, run:
+{{< code shell >}}
+sudo systemctl daemon-reload
+{{< /code >}}
 
+3. Restart the Sensu agent:
 {{< code shell >}}
 sudo service sensu-agent restart
 {{< /code >}}
 
-To restart the Sensu backend, run:
-
+4. Restart the Sensu backend:
 {{< code shell >}}
 sudo service sensu-backend restart
 {{< /code >}}
 
-To confirm the installed version, run `sensu-agent version`, `sensu-backend version`, and `sensuctl version`.
+5. Run a single upgrade command on one your Sensu backends to migrate the cluster:
+{{< code shell >}}
+sensu-backend upgrade
+{{< /code >}}
+
+   To skip confirmation and immediately run the upgrade command, add the `--skip-confirm` flag:
+{{< code shell >}}
+sensu-backend upgrade --skip-confirm
+{{< /code >}}
+
+   {{% notice note %}}
+   **NOTE**: If you are deploying a new Sensu cluster rather than upgrading from a previous version, you do not need to run the `sensu-backend upgrade` command.
+{{% /notice %}}
+
+6. Enter `y` or `n` to confirm if you did *not* add the `--skip-confirm` flag in step 5.
+Otherwise, skip this step.
+
+7. Wait a few seconds for the upgrade command to run.
+
+   You may notice some inconsistencies in your entity list until the cluster finishes upgrading.
+Despite this, your cluster will continue to publish standard check requests and process events.
+
+   If you run the upgrade command more than once, it will not harm the cluster &mdash; you'll just see a response that the upgrade command has already been run.
```

Comment:
I'd like to add a sentence or two here.

"For some minor version upgrades, there will be no database-specific changes, so the `sensu-backend upgrade` tool will report that nothing has been upgraded. Check the release notes for the intermediate versions to see if this is expected or not.